Assembly-CSharp/Global/Sound/VoicePlayer.cs

```diff
@@ -72,19 +72,7 @@ public static void PlayBattleVoice(int va_id, string text)
 
 	public static void PlayBattleScriptVoice(int cmd_id, string text)
 	{
-		string vaPath = String.Format("Voices/{0}/battle/{1}/cmd/va_{1}", Localization.GetSymbol(), FF9StateSystem.Battle.battleMapIndex, cmd_id);
-		if (!String.IsNullOrEmpty(AssetManager.SearchAssetOnDisc("Sounds/" + vaPath + ".akb", true, true)) || !String.IsNullOrEmpty(AssetManager.SearchAssetOnDisc("Sounds/" + vaPath + ".ogg", true, false)))
-		{
-			SoundLib.VALog(String.Format("Voices/{0}/battle/{1}/cmd/va_{1}", Localization.GetSymbol(), FF9StateSystem.Battle.battleMapIndex, cmd_id));
-		}
-		else
-		{
-			// otherwise check for a general one
-			vaPath = String.Format("Voices/{0}/battle/general/va_{1}", Localization.GetSymbol(), cmd_id);
-			if (!String.IsNullOrEmpty(AssetManager.SearchAssetOnDisc("Sounds/" + vaPath + ".akb", true, true)) || !String.IsNullOrEmpty(AssetManager.SearchAssetOnDisc("Sounds/" + vaPath + ".ogg", true, false)){
-				SoundLib.VALog(String.Format("Voices/{0}/battle/general/cmd/va_{1}", Localization.GetSymbol(), cmd_id));
-			}
-		}
+		string vaPath = String.Format("Voices/{0}/battle/cmd/va_{1}", Localization.GetSymbol(), cmd_id);
```

Comment:
Any particular reason, why you choose `String.Format` over interpolated string?

Reply:
Yeah, String. The format is safer and has less chance of causing a conversion failure that could result in an expectation. I never build code expecting everyone else code is perfect.

Reply:
> Yeah, String.Format is safer and has less chance of causing a conversion failure that could result in an expectation.

[Interpolated strings are typically converted into `String.Format` during compliation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated#compilation-of-interpolated-strings). Therefor, typically, interpolated strings are as safe as `String.Format` in that regard.

Reply:
oh it's a compiler optimization I did not know had happened when I learned C# it did not do that and used to get expectation, so i have really just removed a little bit of load from the one of the compilers passes :D